tests/cli/test_cli.py

```diff
@@ -1715,6 +1721,12 @@ def test_search_time(ctx):
 
 @TestModule.register('run')
 def test_run(ctx):
+    # Test thaat bundle fails when run without sufficient time quota
+    _run_command([cl, 'uedit', 'codalab', '--time-quota', '2'])
+    uuid = _run_command([cl, 'run', 'sleep 100000'])
+    wait_until_state(uuid, State.KILLED)
```

Comment:
Does `wait_until_state` take a timeout argument? if so, we might want to pass in a timeout of maybe ~1min

Reply:
Done.